### DISCUSSION TITLE
Permissions

### DISCUSSION THREAD

**Discussion by lazerusrm** [rustdesk/rustdesk-server-pro#456] [General]
So i feel like an idiot a bit when using this software.  I am a pro user with our own server setup and devices added. New To R.D.

Why cant i do the following:

Create a device group, and assign users to access that group. 

I.E. Remote support staff have specific access to group of clients, admins above them have access to all, cross group access according to access policy.

Im the administrator and as soon as i assign a device to a specific user, i loose 100% access, and cant grant them access, and cant grant other users access. Whats the point of groups, permissions etc?

Why is "strategies" assigned per user and not per group? This seems to not make sense to me, as there are plenty of scenarios where if they had control of their own machines, they should have increased permissions, but on other machines, their permissions should be limited.

Why is this made in such a way that is so non-intuitive?

What am i missing? Ive been staring at the documentation and other stuff for hours wondering why im feeling insane

Edit: Ive tried the cross group access permissions, but none of it seems to work.

I assign devices to a user, which makes them a part of a group.

I assign my group to have permissions to control that group.

Client says "The connection is not allowed. The remote user is disabled"

**Comment by rustdesk** [comment]
Could you double check "The remote user is disabled"? You can go to the users page of web console, check the status of the remote  user which is binded to the remote (controlled) device.

**Comment by rustdesk** [comment]
@21pages follow it up for further query.

**Comment by lazerusrm** [comment]
So currently, the primary user i wish to assign, has not created their account for these devices. could it be because their status is "Pending Invite"  

Even though as an admin i should have control? Perhaps this is the edge case here that im experiencing.

But regardless as an Admin, i should be able to bypass this check, perhaps it is a small bug, and the source of my insanity

![pending](https://github.com/user-attachments/assets/b46397da-24f5-4413-88e5-a7457a00eacf)

Ive been trying to inspect the local logs after making connection attempts on my LXC container, but cant seem to find any entries about it in hbbr or hbbs logs.

**Comment by lazerusrm** [comment]
Confirmed edge case.

Recreated user and removed the invite. Success.
